workspaces/tech-insights/plugins/tech-insights-maturity/src/components/MaturityRankInfoCard/MaturityRankInfoCard.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { InfoCard } from '@backstage/core-components';
+import {
+  MaturitySummary,
+  Rank,
+} from '@backstage-community/plugin-tech-insights-maturity-common';
+import { makeStyles } from '@mui/styles';
+import Typography from '@mui/material/Typography';
+import CardContent from '@mui/material/CardContent';
+import Divider from '@mui/material/Divider';
+import Stack from '@mui/material/Stack';
+import { MaturitySummaryCardContent } from '../MaturitySummaryInfoCard/MaturitySummaryCardContent';
+import { MaturityHelp } from '../../helpers/MaturityHelp';
+import { MaturityRankAvatar } from '../MaturityRankAvatar';
+
+type Props = {
+  summary: MaturitySummary;
+};
+
+const useStyles = makeStyles({
+  content: {
+    justifyContent: 'center',
+    margin: 'auto',
+  },
+});
+
+const RankDescription = new Map<number, string>([
```

Comment:
Should these be injected from somewhere like config instead of hardcoding them in here?

Reply:
Ideally yes. But for this release, either I leave it as static or just remove it...